prow/plugins/lgtm/lgtm.go

```diff
@@ -204,3 +207,26 @@ func handle(gc githubClient, log *logrus.Entry, e *event) error {
 	}
 	return nil
 }
+
+func handlePullRequest(gc githubClient, pe github.PullRequestEvent) error {
+	if pe.PullRequest.Merged {
+		return nil
+	}
+
+	switch pe.Action {
+	case "synchronize":
+		// Don't bother checking if it has the label...it's a race, and we'll have
+		// to handle failure due to not being labeled anyway.
+		if err := gc.RemoveLabel(
+			pe.PullRequest.Base.Repo.Owner.Login,
+			pe.PullRequest.Base.Repo.Name,
+			pe.PullRequest.Number,
+			lgtmLabel,
+		); err != nil {
+			// TODO(nlandolfi): inspect this error?
```

Comment:
The way to inspect errors in go is with a type assertion. Check out how it's done with the [assign](https://github.com/kubernetes/test-infra/blob/b4330c6aef47a5f9d99d881439a08333774c297a/prow/github/client.go#L519-L554) method, and how it's used in the [assign plugin](https://github.com/kubernetes/test-infra/blob/b4330c6aef47a5f9d99d881439a08333774c297a/prow/plugins/assign/assign.go#L169-L182). The only tricky thing is that prow's github client retrys 404s a couple times, so we'll need some special casing or something to make that not happen.